ansible/setup.yml

```diff
@@ -6,14 +6,20 @@
   - name: find the ip of docker-machine
     local_action: shell "docker-machine" "ip" "{{docker_machine_name | default('whisk')}}"
     register: result
-    ignore_errors: true
+    when: "'environments/mac' in inventory_dir"
```

Comment:
if we can merge `local` and `mac` such 'cheating' extra work is not needed anymore ;)